reactivesocket-tck-drivers/src/main/java/io/reactivesocket/tckdrivers/server/JavaServerDriver.java

```diff
@@ -129,14 +129,16 @@ public RequestHandler parse() {
                 s.onSubscribe(new TestSubscription(pm));
                 new ParseThread(pm).start();
             }
-        }).withRequestChannel(payloadPublisher -> s -> { // design flaw
+        }).withRequestChannel((payload, payloadPublisher) -> s -> { // design flaw
             try {
                 TestSubscriber<Payload> sub = new TestSubscriber<>();
                 payloadPublisher.subscribe(sub);
                 // want to get equivalent of "initial payload"
                 //sub.request(1); // first request of server is implicit, so don't need to call request(1) here
-                sub.awaitAtLeast(1);
-                Tuple<String, String> initpayload = new Tuple<>(sub.getElement(0).getK(), sub.getElement(0).getV());
+                //sub.awaitAtLeast(1);
```

Comment:
This code looks pretty old, and it turns out I'm wrong about this, the empty constructor for TestSubscriber does an initial request. I will push updates.